docs/UserGuide.md

```diff
@@ -138,8 +139,7 @@ ___
   <br/>e.g. <code>&nbsp; list</code> and <code>list &nbsp;</code> will both be registered as `list`
 
 * Order of prefixed parameters do not matter:
-  <br/> e.g. `add n/NAME e/EPISODE` will obtain the same result as `add e/EPISODE n/NAME`
-  <br/> e.g. `genre c/add 1 g/comedy` will not work as `1` is not a prefixed parameter
+  <br/> e.g. `add n/NAME e/EPISODE` will obtain the same result as `add e/EPISODE n/NAME`. However, `genre c/add 1 g/comedy` will not work as `1` is not a prefixed parameter
```

Comment:
Why combine them tho, i think separating them would be better

Reply:
combined bc the first example illustrates the order of the prefixed params dont matter, but the second example illustrates that `1` is not a prefixed parameter. tbh i dont thik the second point qualifies as an example but i didnt want to remove it so i just added it under the first point as an afterthought